backend/models.py

```diff
@@ -359,6 +360,7 @@ def get_code(self):
     def to_dict(self, include_related=False):
         tmp = serializers.model_to_dict(self, include_related=include_related)
         tmp['code'] = self.get_code()
+        tmp['uri'] = url_for('resource_list', resource='bill', resource_id=self.id, _external=True)
```

Comment:
I think it makes more sense to call this field `url` instead of `uri`, since it's the location of the entity, not the identity.